pkg/kgo/config.go

```diff
@@ -1317,6 +1319,22 @@ func RequireStableFetchOffsets() GroupOpt {
 	return groupOpt{func(cfg *cfg) { cfg.requireStable = true }}
 }
 
+// AdjustFetchOffsetsFn sets the function to be called when a group is joined
+// after offsets are fetched for those partitions so that a user can adjust them
+// before consumption begins.
+//
+// This function combined should not exceed the rebalance interval. It is possible
+// for the group, immediately after finishing a balance, to re-enter a new balancing
+// session. This function is passed a context that is canceled if the current group
+// session finishes (i.e., after revoking).
+//
+// If you are resetting the position of the offset, you may want to clear any existing
+// "epoch" with WithEpoch(-1). If the epoch is non-negative, the client performs
+// data loss detection, which may result in errors and unexpected consuming offsets.
```

Comment:
`unexpected consume offsets`? Probably another typo of mine; I wrote that doc up while cross referencing some of the code, too much window flipping.

Reply:
https://github.com/twmb/franz-go/pull/120/commits/f5414b2e8e0bc08109d6763fb6373052176c6ba0